### PR TITLE
Insurance deductible

### PR DESCRIPTION
**Pull request by peterlangmar**
- we have been breaking devices but the replacment process was unclear

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,9 @@ NOT available for part time or during probation time.
 - Laptop. Preferably with 3 year, on site warranty. Due price-value balance and short warranty, Mac only if it is worth due productivity or a must. If an employee has a good enough laptop, the company could buy it on market price, if it financially makes sense, decided case by case.
 - External screen.
 
+### Policy: Insurance deductible
+- Company covers all devices with an insurance plan. In case somebody: physically breaks breaks, loosest or gets stolen a device outside the office than that person needs to pay the insurance deductible.
```
Comment (cassus) [inline README.md]: "breaks breaks"

**Comment by cassus** [comment]
What's the amount of the deductible compared to the total value?

**Comment by peterlangmar** [comment]
Depends on the final conditions of the insurance, which I still dont have. For Brickflow it was 25k HUF.

**Comment by cassus** [comment]
So this policy is void until we have the insurance contract?

**Comment by peterlangmar** [comment]
No. The amout does bot impact the logic.

**Comment by cassus** [comment]
So how should we apply this policy before we have the insurance contract?

**Comment by peterlangmar** [comment]
The tension is existent and the policy gives an answer to people being in
the situation. We are already applying this just it is not transparent.

**Comment by cassus** [comment]
How are we applying it now without the insurance contract?

**Comment by peterlangmar** [comment]
yes

**Comment by cassus** [comment]
this conversation is broken. let's talk in person

**Comment by peterlangmar** [comment]
@cassus beside the policy I've added a domain to the biz role

**Comment by peterlangmar** [comment]
accepted but merge pending

**Pull request merged**